cfme/scripting/appliance.py

```diff
@@ -35,10 +35,15 @@ def main():
 @main.command('upgrade', help='Upgrades an appliance to latest Z-stream')
 @click.argument('appliance-ip', default=None, required=False)
 @click.option('--cfme-only', is_flag=True, help='Upgrade cfme packages only')
-@click.option('--update-to', default='5.9.z', help='Must be 5.8.z or (5.9.z is default)')
+@click.option('--update-to', default='5.9.z', help='Supported versions 5.8.1, 5.8.2, 5.8.z,'
+              ' 5.9.1, 5.9.2, 5.9.z (.z means latest and default is 5.9.z)')
 def upgrade_appliance(appliance_ip, cfme_only, update_to):
     """Upgrades an appliance"""
-    supported_version_repo_map = {'5.8.z': 'update_url_58', '5.9.z': 'update_url_59'}
+    supported_version_repo_map = {
+        '5.8.z': 'update_url_58', '5.8.0': 'update_url_580', '5.8.1': 'update_url_581',
```

Comment:
**SUGESTION:**  This information and the information on 38 could probably go into cfme_data.yaml.   It's information that will change often and probably should be in a config file rather than hard coded here.

Reply:
What exactly do you suggest for this? I have created sections for current/future releases here, https://gitlab.cee.redhat.com/cfme-qe/cfme-qe-yamls/merge_requests/483/diffs

In terms of it changing often is not the problem exactly, the whole migration procedure could change for each build so that is going to potentially need updating regardless of the build versions.

Reply:
The actual versions should be in the config such that the whole supported_version_repo_map is simply read in from config.   Does that make sense?

Reply:
This does make sense, I think though it can be a secondary PR